process/block/sovereignChainBlock.go

```diff
@@ -792,6 +788,16 @@ func (scbp *sovereignChainBlockProcessor) ProcessBlock(headerHandler data.Header
 		return nil, nil, err
 	}
 
+	newBody := body
+	defer func() {
+		if err != nil {
```

Comment:
this if does not have a reason to be here.

Reply:
You are absolutely right here, deleted it.